src/redis-cli.c

```diff
@@ -2563,6 +2569,7 @@ int main(int argc, char **argv) {
 
     config.hostip = sdsnew("127.0.0.1");
     config.hostport = 6379;
+    config.connect_timeout = (struct timeval){ .tv_sec = 60 };
```

Comment:
Are you sure we need to cast to (struct timeval) here in order to initialize the structure?

Reply:
I'm totally open to alternatives - would `config.connect_timeout.tv_sec = 60;` make more sense?